### PR TITLE
updated to work with current versions of vue and summernote

### PR DESCRIPTION
**Pull request by jclyons52**

### CODE REVIEW

**Comment by jclyons52** [comment]
these changes break backwards compatibility

File: README.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ The HTML snippets are as follows:
 
 ```html
 <div class="container" id="app">
-  <vue-html-editor model="{{@ text}}"></vue-html-editor>
+  <vue-html-editor  :model.sync="text"></vue-html-editor>
   <div style="margin-top:40px">
     <div> The HTML contents are as follows:</div>
     <hr>
-    <div >{{{text}}}</div>
+    <div >{{text}}</div>
```
Comment (jclyons52) [inline README.md:46]: The change to {{{ text }}} is not needed, and perhaps should be left as it was

**Comment by maciao** [comment]
Thanks you that is correct. @jclyons52

**Comment by Haixing-Hu** [comment]
Upgrade to the Vue.js v1.0.x

**Pull request closed without merging**